OpenRA.Mods.Common/Traits/SupportPowers/SelectDirectionalTarget.cs

```diff
@@ -60,27 +64,27 @@ IEnumerable<Order> IOrderGenerator.Order(World world, CPos cell, int2 worldPixel
 
 			if (mi.Button == MouseButton.Left && mi.Event == MouseInputEvent.Down)
 			{
-				if (!beginDrag)
+				if (!activated)
 				{
 					targetCell = cell;
-					location = mi.Location;
-					beginDrag = true;
+					targetLocation = mi.Location;
+					activated = true;
+					Game.HideCursor = true;
```

Comment:
Can reduce the invisible cursor blink by setting this before returning from `RenderAboveShroud`.